samples/nrf_desktop/src/services/hids.c

```diff
@@ -200,6 +225,23 @@ static int module_init(void)
 		ir_pos++;
 	}
 
+
+
+	/* Configuration channel feature report */
```

Comment:
this comment is really excessive (note that configuration option tells already what this part is about